tests/unit/submitter/test_ena_submitter.py

```diff
@@ -0,0 +1,191 @@
+                'utf-8'
+            )
+        ])
+        archive_responses = self.submitter.send_all_entities(converted_ena_study, self.ARCHIVE_TYPE)
+
+        assert_that(len(archive_responses)).is_equal_to(len(converted_ena_study))
+        for archive_response in archive_responses:
+            assert_that(archive_response.data).is_not_none()
+            assert_that(archive_response.entity_type).is_equal_to(self.entity_type)
+            assert_that(archive_response.is_update).is_equal_to(is_update)
+
+    def test_processing_archive_response_from_project_creation_add_result_for_created(self):
+        payload = {'accession': 'ERP12345', 'uuid': self.project_uuid}
+        archive_response = create_archive_response(payload, self.entity_type)
+        archive_responses = [archive_response]
+        processed_responses_from_archive =\
+            self.submitter.process_responses(self.submission, archive_responses, '', self.ARCHIVE_TYPE)
+
+        assert_that(len(processed_responses_from_archive)).is_equal_to(1)
+        assert_that(processed_responses_from_archive).contains('CREATED')
+        assert_that(processed_responses_from_archive.get('CREATED')).is_equal_to(archive_responses)
+
+    def test_processing_archive_response_from_project_update_add_result_for_updated(self):
+        payload = {'accession': 'ERP12345', 'uuid': self.project_uuid}
+        archive_response = create_archive_response(payload, self.entity_type, is_update=True)
+        archive_responses = [archive_response]
+        processed_responses_from_archive =\
+            self.submitter.process_responses(self.submission, archive_responses, '', self.ARCHIVE_TYPE)
+
+        assert_that(len(processed_responses_from_archive)).is_equal_to(1)
+        assert_that(processed_responses_from_archive).contains('UPDATED')
+        assert_that(processed_responses_from_archive.get('UPDATED')).is_equal_to(archive_responses)
+
+    def test_processing_archive_response_from_sample_update_add_result_for_updated(self):
+        payload = {'error_messages': ['Dummy error message'], 'accession': 'ERP12345', 'uuid': self.project_uuid}
+        archive_response = \
+            create_archive_response(payload, self.entity_type, is_update=True)
+        archive_responses = [archive_response]
+        processed_responses_from_archive =\
+            self.submitter.process_responses(self.submission, archive_responses, '', self.ARCHIVE_TYPE)
+
+        assert_that(len(processed_responses_from_archive)).is_equal_to(1)
+        assert_that(processed_responses_from_archive).contains('ERRORED')
+        assert_that(processed_responses_from_archive.get('ERRORED')).is_equal_to(archive_responses)
+
+    @staticmethod
+    def __get_expected_payload(filename: str):
+        with open(dirname(__file__) + filename) as file:
+            return file.read()
+
+
+    # def test_submit_project_converts_and_sends_project(self):
```

Comment:
IS this commented code on the way out, or about to be fixed?

Reply:
Accidentally forgot to delete it. It is done now. Tx